src/components/Events/Fall_GM/Fall_GM.js

```diff
@@ -5,57 +5,196 @@ import Countdown from 'react-countdown';
+				<Banner decorative />
+				<Countdown date={FALL_GM_2020_START_TIME} renderer={renderCountdownOrNow} />
+			</div>
+		);
+	}
+}
+
+class GM_NOW extends React.Component{
+	renderCommitteeSpeakers() {
+		const committeeData = [
+			{
+				id: "studio",
+				displayName: "ACM Studio",
+				speakers: "Kade Adams, Evan Charfauros",
+				logo: "/images/committees/logo-studio.svg",
+			},
+			{
+				id: "icpc",
+				displayName: "ACM ICPC",
+				speakers: "Nikil Selvam",
+				logo: "/images/committees/logo-icpc.svg",
+			},
+			{
+				id: "design",
+				displayName: "ACM Design",
+				speakers: "Haki Alpay, Tomoki Fukazawa",
+				logo: "/images/committees/logo-design.svg",
+			},
+			{
+				id: "cyber",
+				displayName: "ACM Cyber",
+				speakers: "Disha Zambani, Ellie Cheng",
+				logo: "/images/committees/logo-cyber.svg",
+			},
+			{
+				id: "teach",
+				displayName: "ACM TeachLA",
+				speakers: "Matt Wang",
+				logo: "/images/committees/logo-teachla.svg",
+			},
+			{
+				id: "w",
+				displayName: "ACM W",
+				speakers: "Julia Baylon",
+				logo: "/images/committees/logo-w.svg",
+			},
+			{
+				id: "ai",
+				displayName: "ACM AI",
+				speakers: "John Dang",
+				logo: "/images/committees/logo-ai.svg",
+			},
+			{
+				id: "hack",
+				displayName: "ACM Hack",
+				speakers: "Raji Jadhav",
+				logo: "/images/committees/logo-hack.svg",
+			},
+		]
+		return committeeData.map((committee) => {
 			return (
-				<div id="time-to-gm-wrapper">
-					<h1 id="time-to-gm-headline">Fall General Meeting 2020 begins in</h1>
-					<div id="countdown-wrapper">
-						<div className="countdown-cards">
-							<div className="square-background"/>
-							<div className="countdown-numbers">{days}</div>
-							<div className="countdown-labels">{dayString}</div>
+			<p><span id={committee.id}><img src={process.env.PUBLIC_URL + committee.logo} alt={"Logo for " + committee.displayName} className="committee-logos"/>{committee.displayName}</span>: {committee.speakers}</p>
+			);
+		})
+	}
+
+	render () {
+		return (
+			<div id="gm-now-wrapper">
+				<h1 id="time-to-gm-headline-now">Fall General Meeting 2020 is happening right now!</h1>
+				<div id="gm-main-content">
+					<div id="join-links-wrapper">
+						<div className="click-to-join">
+							<a href="http://example.com/" target="_blank" rel="noopener noreferrer">
+								<div id="zoom-join-link-wrapper" className="join-links">
+									<img src={process.env.PUBLIC_URL + "/images/ZoomLogo.png"} alt="Zoom logo" className="join-links-img"/>
+								</div>
+							</a>
+							<p className="join-us">Join us from 6:30 to 7:10 p.m. PDT in Zoom</p>
 						</div>
-						<div className="countdown-cards">
-							<div className="square-background"/>
-							<div className="countdown-numbers">{hours}</div>
-							<div className="countdown-labels">{hourString}</div>
+						<div className="click-to-join">
+							<a href="https://bit.ly/ACMdiscord" target="_blank" rel="noopener noreferrer">
```

Comment:
Slightly unrelated, but could we make the discord link a constant somewhere? I know it's out of scope of this PR.

Reply:
There is technically a separate social media component but it has all of the social media at once, and are not individually available for cases like this. Good point, though! If we don't get a chance to address it here, I'll add it as a task to the dev team backlog

Reply:
Will resolve conversation now, but will seek to address this in a later PR.